capabilities/lib/authservice/secretBuilder.ts

```diff
@@ -48,12 +48,14 @@ export class AuthServiceSecretBuilder {
       const name = this.decodeBase64(secret, "name");
       const domain = this.decodeBase64(secret, "domain");
       const id = this.decodeBase64(secret, "id");
+      const realm = this.decodeBase64(secret, "realm");
       return AuthserviceConfig.createSingleChain({
         id,
         name,
         hostname: `${name}.${domain}`,
         redirect_uri: this.decodeBase64(secret, "redirect_uri"),
         secret: this.decodeBase64(secret, "secret"),
+        logout_redirect_uri: `https://keycloak.${domain}/auth/realms/${realm}/protocol/openid-connect/logout`,
```

Comment:
You want to also include the client id in the logout URI:
```suggestion
        logout_redirect_uri: `https://keycloak.${domain}/auth/realms/${realm}/protocol/openid-connect/logout/?client_id=${id}`,
```

If you view your user's sessions on keycloak in the browser, you should see the name of this client go away if the logout was successful